src/test/java/net/openhft/chronicle/queue/impl/single/ExcerptsPerCycleTestOurs.java

```diff
@@ -0,0 +1,114 @@
+package net.openhft.chronicle.queue.impl.single;
+import net.openhft.chronicle.core.io.IOTools;
+import net.openhft.chronicle.queue.ExcerptTailer;
+import net.openhft.chronicle.queue.RollCycle;
+import net.openhft.chronicle.queue.impl.single.SingleChronicleQueue;
+import net.openhft.chronicle.queue.impl.single.SingleChronicleQueueBuilder;
+import net.openhft.chronicle.wire.DocumentContext;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.StreamCorruptedException;
+import java.nio.file.Files;
+import java.util.concurrent.BrokenBarrierException;
+import java.util.concurrent.CyclicBarrier;
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.function.LongConsumer;
+
+public final class ExcerptsPerCycleTestOurs {
```

Comment:
I don't think this name will get picked up by surefire?

Reply:
I'm not sure it will be merged under this name, but new JUnit is annotations-based and it is present.

Reply:
I was referring to surefire inclusion, does that change when you're using Jupiter somehow?
https://maven.apache.org/surefire/maven-surefire-plugin/examples/inclusion-exclusion.html